File: pset6/mario less/mario.py
from cs50 import get_int

# Prompting the height from the user
while True:
    height = get_int("A number between 1 and 8: ")
    if height in range(1, 9):
        break

# Printing the pyramid
for printing in range(height):
    print((height - printing - 1) * " ", end="")
    print((printing + 1) * "#")

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Get int from user
    int i;
    do
    {
        i =  get_int("A number between 1 and 8: ");
    }
    while (i < 1 || i > 8);

    //Print the "#" and " ".
    for (int h = 0 ; h < i ; h++)
    {
        for (int x = i - h ; x > 1; x--)
        {
            printf(" "); // The spaces
        }
        for (int l = 0 ; l < h + 1 ; l++)
        {
            printf("#"); // The hashes
        }
        printf("\n");
    }
}

